internal/tiltfile/tiltfile_test.go

```diff
@@ -814,6 +815,30 @@ k8s_resource('foo', links=EXPR)
 				db(image("gcr.io/foo")),
 				deployment("foo"))
 		})
+
+		t.Run("dc-"+c.name, func(t *testing.T) {
+			f := newFixture(t)
+			defer f.TearDown()
+
+			f.setupFoo()
```

Comment:
this sets up a kubernetes file. i think you need a docker_compose file? something like

```
	f.file("docker-compose.yml", `version: '3'
services:
  foo:
    image: gcr.io/foo
`)
```

and then drop

```
docker_compose('docker-compose.yml')
```

next to dc_resource

Reply:
Will do! Thanks for taking a look

Reply:
Got the tests appropriately setup (and more importantly passing).